docs/tutorials/manifest-parser.md

```diff
@@ -75,8 +75,8 @@ destroyed.
 #### registration
 
 At the end of the file, you should register the parser with the library.  This
-will allow it to be used by the `Player`.  There are two methods:
-`registerParserByExtension` and `registerParserByMime`.  They both add parsers
+will allow it to be used by the `Player`.  There is one method:
+`registerParserByMime`.  They both add parsers
```

Comment:
"Use `registerParserByMime` to add the parser to the registry of manifest parsers."

The sentence "they both add parsers" is no longer applicable.

Reply:
Done!